backend/src/util.rs

```diff
@@ -180,6 +180,25 @@ pub struct Price {
     pub price_pupil: u32,
 }
 
+/// The nutrients of a dish
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct NutritionData {
+    /// Energy in Kcal
+    pub energy: u32,
+    /// Protein in grams
+    pub protein: u32,
+    /// Carbs in grams
+    pub carbohydrates: u32,
+    /// Sugar in grams
+    pub sugar: u32,
+    /// Fat in grams
+    pub fat: u32,
+    /// Saturated fat in grams
+    pub saturated_fat: u32,
+    /// Salt in grams
+    pub salt: u32,
+}
+
```

Comment:
why is that here? is that not sufficient in the model?

Reply:
Because this is used throughout the backend

Reply:
I remember we had talked about that. My oddity-detection is just triggered that environment info is not her but this is.

Reply:
Environment_info was here, but i moved it to two different places, because there are two variants: One with Average Rating (Pre-Database) and one without (post-database)

Reply:
yea, i see

Reply:
all these duplicate similar structs are a bit a problem of our code base but i still think they should be seperate structs if they specify seperate things. Maybe this could be solved using a variant of the typestate pattern in another life.
(This would allow **one** struct definition to produce multiple distinct types---one with an attribute and one without---that can be differentiated at compile time)